specs/support/schema_statement_examples.rb

```diff
@@ -133,55 +134,72 @@ def self.execute(query); query; end
     let(:view_name) { :view_name }
 
     describe "view_name" do
-      it "takes a symbol as the view_name" do 
-        shared_base.drop_view(view_name.to_sym).must_match(/#{view_name}/)
+      it "takes a symbol as the view_name" do
+        _(shared_base.drop_view(view_name.to_sym)).must_match(/#{view_name}/)
       end
 
-      it "takes a string as the view_name" do 
-        shared_base.drop_view(view_name.to_s).must_match(/#{view_name}/)
+      it "takes a string as the view_name" do
+        _(shared_base.drop_view(view_name.to_s)).must_match(/#{view_name}/)
       end
     end
   end
 
+  describe "#tables" do
```

Comment:
These new tests were helpful in diagnosing the problems arising from the incompatible results coming from the mysql jdbc adapter.

Reply:
🤘